index.html

```diff
@@ -0,0 +1,611 @@
+                <span class="icon-apple2"></span><br>
+                <p>iOS</p>
+              </a>
+            </div>
+          </aside>
+        </div>
+        <img src="/img/portrait-display-mockup.png" alt="" title="">
+      </section>
+      <section data-section="why-appbox" id="why">
+        <h1 class="header-underline">Why Choose AppBox</h1>
+        <img src="/img/iphone-6.png" alt="Image of iPhone" title="iPhone">
+        <ul class="why-1">
+          <li>
+            <div class="why-icon-wrapper">
+              <span class="icon-cubes"></span>
+            </div>
+            <div class="why-item-wrapper">
+              <h2>App store</h2>
+              <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+            </div>
+          </li>
+          <img src="/img/spacer-dots.svg">
+          <li>
+            <div class="why-icon-wrapper">
+              <span class="icon-screen"></span>
+            </div>
+            <div class="why-item-wrapper">
+              <h2>Retina ready</h2>
+              <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+            </div>
+          </li>
+          <img src="/img/spacer-dots.svg">
+          <li>
+            <div class="why-icon-wrapper">
+              <span class="icon-apple"></span>
+            </div>
+            <div class="why-item-wrapper">
+              <h2>Premium support</h2>
+              <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+            </div>
+          </li>
+        </ul>
+        <ul class="why-2">
+          <li>
+            <div class="why-icon-wrapper">
+              <span class="icon-flag"></span>
+            </div>
+            <div class="why-item-wrapper">
+              <h2>Multipurpose layout</h2>
+              <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac <u></u>llamcorper fusce</p>
+            </div>
+          </li>
+          <img src="/img/spacer-dots.svg">
+          <li>
+            <div class="why-icon-wrapper">
+              <span class="icon-bubble"></span>
+            </div>
+            <div class="why-item-wrapper">
+              <h2>Live chat support</h2>
+              <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac <u></u>llamcorper fusce</p>
+            </div>
+          </li>
+          <img src="/img/spacer-dots.svg">
+          <li>
+            <div class="why-icon-wrapper">
+              <span class="icon-coffee"></span>
+            </div>
+            <div class="why-item-wrapper">
+              <h2>And many more</h2>
+              <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+            </div>
+          </li>
+        </ul>
+      </section>
+      <section data-section="app-reviews">
+        <div class="review-icon">
+          <span class="icon-quote"></span>
+        </div>
```

Comment:
Again with the icon wrapping markup. Watch out for this anti-pattern, and be aware of unnecessary markup and classes; collapse and combine whenever possible.